src/composables/useGoalData.js

```diff
@@ -175,6 +175,65 @@ export default function useGoalData({ apollo }) {
 		loading.value = false;
 	}
 
+	/**
+	 * Replaces all goals in user preferences with newGoals
+	 * @param {*} newGoals - Array of new goal objects to set
+	 */
+	async function replaceAllGoals(newGoals) {
+		if (!userPreferences.value?.id) {
+			await createUserPreferences(apollo, { goals: [] });
+			await loadPreferences('network-only');
+		}
+		const parsedPrefs = JSON.parse(userPreferences.value?.preferences || '{}');
+		parsedPrefs.goals = newGoals;
+		await updateUserPreferences(apollo, userPreferences.value, parsedPrefs, { goals: newGoals });
+		setGoalState({ goals: newGoals });
+	}
+
+	/**
+	 * This method shows Goal Entry for 2026 Goals
+	 * It invalidates all goals on Jan 1st of 2026
+	 * @return {Array} - expiredGoals
+	 */
+	async function renewAnnualGoal(today = new Date()) {
+		const parsedPrefs = await loadPreferences();
+		const goals = parsedPrefs.goals || [];
+		let expiredGoals = [];
+
+		if (today.getMonth() === 0 && today.getDate() === 1) {
+			expiredGoals = goals.map(prevGoal => ({
+				...prevGoal,
+				active: false
+			}));
+			await replaceAllGoals(expiredGoals);
```

Comment:
I don't think we want to replace the previous goal, just mark it as complete (or maybe a new status related to never being completed if it wasn't complete).